src/particles/diagnostics/DiagnosticOutput.cpp

```diff
@@ -81,10 +82,23 @@ namespace impactx::diagnostics
                 } // if( otype == OutputType::PrintParticles)
                 else if (otype == OutputType::PrintNonlinearLensInvariants) {
 
-                    amrex::ParticleReal const alpha = 0.0;
-                    amrex::ParticleReal const beta = 1.0;
-                    amrex::ParticleReal const tn = 0.4;
-                    amrex::ParticleReal const cn = 0.01;
+                    using namespace amrex::literals;
+
+                    // Parse the diagnostic parameters
+                    amrex::ParmParse pp_dist("diag");
```

Comment:
Let's document these user-facing parameters in the usual `parameters.rst`, section _Diagnostics_ - _Reduced Diagnostics_
https://impactx.readthedocs.io/en/latest/usage/parameters.html#diagnostics-and-output

Reply:
Just submitted the documentation in a new commit.